barcodes_gs1_label/security/barcodes_gs1_label.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright NuoBiT Solutions - Eric Antones <eantones@nuobit.com>
+     Copyright NuoBiT Solutions - Frank Cespedes <fcespedes@nuobit.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl) -->
+<odoo noupdate="1">
+    <record id="barcodes_gs1_label_company_rule" model="ir.rule">
```

Comment:
Adapt this to https://github.com/OCA/odoo-community.org/blob/2ae9761ef59c215e3964c6767e29e3b10d081fd8/website/Contribution/CONTRIBUTING.rst#L440-L443
https://github.com/OCA/odoo-community.org/blob/2ae9761ef59c215e3964c6767e29e3b10d081fd8/website/Contribution/CONTRIBUTING.rst#L478-L480